app/api/ada/lib/cardanoCrypto/cryptoWallet.js

```diff
@@ -100,3 +100,19 @@ export function getCryptoDaedalusWalletFromMasterKey(
   const wallet = RustModule.WalletV2.DaedalusWallet.new(privateKey);
   return wallet;
 }
+
+/**
+ * Generate catalyst private key for QR code
+ */
+
+export function generatePrivateKeyForCatalyst(): RustModule.WalletV4.Bip32PrivateKey {
```

Comment:
I think this is fine for the first version to get things working, but I would like to make a modification to CIP1852 to include a voting key path instead of generating random keys like this every time going forward.